website/project/model.py

```diff
@@ -3436,13 +3415,6 @@ class Sanction(StoredObject):
     # Sanction subclasses must have an initiated_by field
     # initiated_by = fields.ForeignField('user', backref='initiated')
 
-    # Expanded: Dictionary field mapping admin IDs their approval status and relevant tokens:
```

Comment:
It was in error. I thought that was the expanded property from the node that the old project organizer used. My guess is they copied the comment from said property and didn't rename it, because it's probably supposed to be called `approval_state`.